experimental/msal-react/.prettierignore

```diff
@@ -0,0 +1 @@
+src/index.ts
```

Comment:
Prettier was having issues with `export type`, which for some reason wasn't being resolved with an ignore statement.